PYME/LMVis/layers/image_layer.py

```diff
@@ -272,7 +272,7 @@ def update_from_datasource(self, ds):
         
         if not self._im_key == im_key:
             self._im_key = im_key
-            self._im = ds.data[:,:,self.slice, self.channel].astype('f4').squeeze()# - c0)/(c1-c0)
+            self._im = ds.data_xyztc[:,:,self.slice,:,self.channel].astype('f4').squeeze()# - c0)/(c1-c0)
```

Comment:
This will break badly if you ever give it 5D data (self._im must be a 2D array). It's probably best to stick with the collapsed z/t for now (potentially explicitly if you want to remove the deprecation warning). A 5D version needs another parameter analogous to slice. 

This probably also makes the changes to the visgui dsviewer plugin are also a bit premature.

Reply:
Will revert